packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -349,7 +349,7 @@ public void onProgress(
     // shared under the hood.
     // See https://github.com/square/okhttp/wiki/Recipes#per-call-configuration for more information
     if (timeout != mClient.connectTimeoutMillis()) {
```

Comment:
This line is no longer correct. I can fix it up quickly.

Reply:
how change this line?